src/main/java/com/bina/lrsim/pb/Spec.java

```diff
@@ -122,6 +141,26 @@ public static Spec fromReadType(final String readType) {
         return UnknownSpec;
     }
 
+    public boolean isPolymeraseReadFlag() {
+        return polymeraseReadFlag;
+    }
+
+    public byte[] getAdapterSequence() {
+        return adapterSequence;
+    }
+
+    public void setPolymeraseReadFlag(final String outputPolymeraseRead) {
+        if (outputPolymeraseRead.equals("True")) {
```

Comment:
this can be a bit dangerous since it's one-sided, let's try to use a ``boolean`` paramter here such that ```this.polymeraseReadFlag = outputPolymeraseRead```